### ISSUE TITLE
Link error

### ISSUE THREAD

**Issue by gregoriopellegrino** [w3c/publ-a11y#386]
As I am trying to publish the draft, doing the tests with the different validators, it turns out that the address http://www.example.com/a11y/report/9780000000001 that we used in the principles does not exist. This causes an error with the W3C checklink validator.

**Comment by mattgarrish** [comment]
You can ignore the error since this won't be published in the TR reports.

But maybe it would be good to put some sort of placeholder document in this repository and link to it (maybe in a /examples directory). That way if anyone tries out the link while reading the principles they won't just land at a 404.

**Comment by gautierchomel** [comment]
IMHO, this should be a link to an example of a certification report so everyone understands what is expected to find in such reports.

**Comment by GeorgeKerscher** [comment]
Since @mattgarrish added the link, I think this should be closed. If I am mistaken, please reopen.
